permissions/plugin-sysdig-secure.yml

```diff
@@ -7,3 +7,4 @@ developers:
 - "alfredlandrum"
 - "sysdig"
 - "jequals5"
+- "sethmccombs"
```

Comment:
Looks like this is not the correct username for the Jenkins infra. Please verify this is the username you use to login to [Artifactory](https://repo.jenkins-ci.org/webapp/#/login).